sqlmesh/core/config/feature_flag.py

```diff
@@ -0,0 +1,9 @@
+from sqlmesh.utils.pydantic import PydanticModel
+
+
+class DbtFeatureFlag(PydanticModel):
+    scd_type_2_support: bool = True
+
+
+class FeatureFlag(PydanticModel):
+    dbt: DbtFeatureFlag = DbtFeatureFlag()
```

Comment:
We want the feature flag to be dbt specific?

Reply:
my guess is that this is a categorization for dbt specific feature flags

Reply:
Yeah this feature flag is dbt specific. If native users start using a new SCD Type 2 feature and they hit an error they can just remove it and not affect prod. In dbt, since we used to ignore snapshots and now we process them, their prod becomes affected and therefore this FF is an easy to reach stability again. Therefore it is dbt focused.